### ISSUE TITLE
I have a question 

### ISSUE THREAD

**Issue by Tree54335** [jerivalera77/test#1]
Can you add my server to this, please? If you can I tell you what it is is.

**Comment by jerivalera77** [comment]
Sure, what is it?

**Comment by qzenlebg** [comment]
hello i wanted to known how did you did to translate eaglercraft java to html file and how the world generation works ? thank you

**Comment by Tree54335** [comment]
Are you saying how i created a world and the website?
________________________________

**Comment by qzenlebg** [comment]
Yes like how dis you translates the eaglercraft file into a website

**Comment by qzenlebg** [comment]
Yes like how dis you translates the eaglercraft file into a website my discord : qzen#4692

**Comment by qzenlebg** [comment]
Yes like how did you translates the eaglercraft file into a website my discord : qzen#4692
